pkg/apis/steward/v1alpha1/run_types.go

```diff
@@ -219,3 +225,11 @@ type PipelineRunDetails struct {
 	// +optional
 	Cause string `json:"cause,omitempty"`
 }
+
+// Profiles defines a set of aspects which are restricted to a named profile
+// for this aspect.
+type Profiles struct {
```

Comment:
`Profiles` or `Profile`?

Reply:
I think Profiles is correct. Currently we only have network profile but more will follow.

Reply:
Ah, OK... "Network Profile" and others in the future.